packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -290,7 +302,7 @@ async function doEmit(mainTspFile: string, emitter: Emitter) {
     return;
   }
   /*Config emitter output dir and emit in tspconfig.yaml. */
-  const defaultEmitOutputDirInConfig = `{project-root}/${emitter.kind}/${getLanguageAlias(emitter.language)}`;
+  const defaultEmitOutputDirInConfig = `{output-dir}/${emitter.package}`;
```

Comment:
variable {emitter-name} doesnt work?

Reply:
we don't have emitter.name meta-data. and emitter name is the same as package name.

Reply:
variable {emitter-name} works. I updated.